.drone.star

```diff
@@ -781,9 +802,10 @@ def localApiTestPipeline(ctx):
                             "steps": skipIfUnchanged(ctx, "acceptance-tests") +
                                      restoreBuildArtifactCache(ctx, "ocis-binary-amd64", "ocis/bin") +
                                      ocisServer(storage, params["accounts_hash_difficulty"], extra_server_environment = params["extraServerEnvironment"]) +
+                                     waitForEmailService(params["emailNeeded"]) +
```

Comment:
```suggestion
                                     waitForEmailService() if params["emailNeeded"] else [] +
```

Reply:
should it be `waitForEmailService(params["emailNeeded"]) if params["emailNeeded"] else []` ? 
`waitForEmailService()` requires a parameter